app/src/main/java/org/smartregister/addo/activity/FamilyProfileActivity.java

```diff
@@ -140,6 +141,13 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 // TODO Add Menu Item here
 
                 break;
+            case android.R.id.home:
+                //onBackPressed();
+                Intent intent = NavUtils.getParentActivityIntent(this);
+                assert intent != null;
```

Comment:
Not sure why the `assert` is used here? Is it an alternative to an` if...else`?

Reply:
Not sure as well but I guess it was an IDE suggestion that was accidentally introduced